src/nidcpower/metadata/enums.py

```diff
@@ -62,6 +62,42 @@
             }
         ]
     },
+    'CurrentLevelAutorange': {
+        'values': [
+            {
+                'documentation': {
+                    'description': 'Autoranging is disabled.'
+                },
+                'name': 'NIDCPOWER_VAL_OFF',
+                'value': 0
+            },
+            {
+                'documentation': {
+                    'description': 'Autoranging is enabled.'
+                },
+                'name': 'NIDCPOWER_VAL_ON',
+                'value': 1
+            }
+        ]
+    },
```

Comment:
The enum values don't match C headers:
```
/*- Defined values for attributes -*/
/*-   NIDCPOWER_ATTR_VOLTAGE_LEVEL_AUTORANGE -*/
/*-   NIDCPOWER_ATTR_CURRENT_LIMIT_AUTORANGE -*/
/*-   NIDCPOWER_ATTR_CURRENT_LEVEL_AUTORANGE -*/
/*-   NIDCPOWER_ATTR_VOLTAGE_LIMIT_AUTORANGE -*/
/*-   NIDCPOWER_ATTR_AUTO_ZERO -*/
#define NIDCPOWER_VAL_OFF                                           (0L)
#define NIDCPOWER_VAL_ONCE                                          (NIDCPOWER_VAL_BASE + 24L)
#define NIDCPOWER_VAL_ON                                            (1L
```

Why?

Reply:
This was added by mistake, I will remove these.